client/src/components/modal-forms/ArchiveHiredParticipantForm.js

```diff
@@ -12,11 +12,50 @@ import {
   archiveTypeOptions,
   UnsuccessfulCohortReason,
   PSIEducationUnderwayStatus,
+  EmploymentEndedType,
+  ROSCompletedType,
+  SuccessfulROSReason,
+  ROSUnderwayStatus,
+  ROSCompleteStatus,
 } from '../../constants';
 import { getTodayDate } from '../../utils';
+import { fetchParticipantReturnOfServiceStatus } from '../../services';
 
-const statusOptions = archiveStatusOptions.map((option) => ({ value: option, label: option }));
-const reasonOptions = archiveReasonOptions.map((option) => ({ value: option, label: option }));
+/**
+ * Returns form values dependent on the rosStatus of the participant
+ * isROS is for logic purposes
+ * @param {number} participantId
+ * @returns { types, date, isROS } => { [{}], string, boolean }
+ */
+const fetchFormData = async (participantId) => {
```

Comment:
**Suggestion**
You can pass the selected participant to ArchivedHiredPartcipantForm. (
Check `actionMenuParticipant`@`ParticipantTable` | `actionMenuParticipant`@ `ParticipantTableDialogues`)
The ros status info is included in participant data. So, you can save an unnecessary remote API call